src/applications/vaos/newAppointmentFlow.js

```diff
@@ -167,15 +167,8 @@ export default {
       if (eligibilityStatus.direct) {
         let appointments = null;
 
-        // If we can't get the history, then continue anyway
-        // and we'll show the full clinic list
```

Comment:
I think this is actually incorrect, we should go to the request flow if we can't get appt history.